dpnp/dpnp_iface_window.py

```diff
@@ -72,16 +72,26 @@ def _call_window_kernel(
     exec_q = result.sycl_queue
     _manager = dpu.SequentialOrderManager[exec_q]
 
-    ht_ev, win_ev = _window_kernel(
-        exec_q, dpnp.get_usm_ndarray(result), depends=_manager.submitted_events
-    )
+    if beta is None:
+        ht_ev, win_ev = _window_kernel(
+            exec_q,
+            dpnp.get_usm_ndarray(result),
+            depends=_manager.submitted_events,
```

Comment:
Do we really need to pass dependent events? It seems windows kernels have no dependency on submitted events.